detox/e2e/test/autocomplete/at_mention.e2e.js

```diff
@@ -167,4 +167,24 @@ describe('Autocomplete', () => {
         // * Expect at mention autocomplete not to contain associated user suggestion
         await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).not.toExist();
     });
+
+    it('should display suggestions in sequence', async () => {
```

Comment:
I wasn't clear on how to add a new test case on Jira, so I haven't been able to add a TM4J id on this yet

Help with this would be greatly appreciated! :)

Reply:
probably @saturninoabril or @josephbaylon can  help with this

Reply:
Sure, I can help with that. Will provide once available -- just need to check if this is already available or if there's related test that needs to be updated, else will create a new one.

Reply:
Added MM-T3409_11 per commit suggestion below